### PR TITLE
BXMSPROD-59 split slave node configuration into roles.

### PR DESCRIPTION
**Pull request by pszubiak**
* kie role - tasks specific to community node configuration.
* rhba role - tasks specific to product node configuration.
* common role - task common to all roles.

Signed-off-by: Piotr Szubiakowski <pszubiak@redhat.com>

### CODE REVIEW

File: jenkins-slaves/ansible/roles/rhba/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+- name: Import RCMTOOLS repository key
+  shell: rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-rcminternal
+
+- name: Install packages
+  yum:
+    pkg: "{{item}}"
+    state: latest
+  with_items:
+   - brewkoji
+   - python2-pip
```
[the remark this answers is not in the document]
Comment (pszubiak) [reply jenkins-slaves/ansible/roles/rhba/tasks/main.yml]: @AntStephenson Do you mean python2-pip package? Or everything?

[the remark this answers is not in the document]
Comment (pszubiak) [reply jenkins-slaves/ansible/roles/rhba/tasks/main.yml]: Do you mean python2-pip package?

[the remark this answers is not in the document]
Comment (AntStephenson) [reply jenkins-slaves/ansible/roles/rhba/tasks/main.yml]: Ah, yes. I wrote a single line comment, but it's hard to see the context.

[the remark this answers is not in the document]
Comment (AntStephenson) [reply jenkins-slaves/ansible/roles/rhba/tasks/main.yml]: python2-pip

[the remark this answers is not in the document]
Comment (pszubiak) [reply jenkins-slaves/ansible/roles/rhba/tasks/main.yml]: Yep, let's move it to common :)

**Pull request merged**